docker/datahub-gms/Dockerfile

```diff
@@ -25,6 +25,11 @@ RUN apk --no-cache --update-cache --available upgrade \
 
 FROM --platform=$BUILDPLATFORM alpine:3.14 AS prod-build
 
+# Add the kinit command to support Kerberos
+RUN apk update
+RUN apk add krb5 \
+    && apk add krb5-server
```

Comment:
Question: Why does the gms container need to have krb installed in it?

Reply:
Kerberos authentication has expiration time, if we set it to one day, then we can not only authenticate once, we need multiple authentication, but we do not need to manually authenticate. After starting the container, it will automatically authenticate, but we need the command of kinit

Reply:
![Uploading GK$_30Q40Q4PO7JW30`2Q0S.png…]()

Reply:
![GK$_30Q40Q4PO7JW30`2Q0S](https://user-images.githubusercontent.com/71497399/179151619-1306fba0-e564-441a-af69-338093ab44c0.png)